source/includes/oai-pmh/_root.md

```diff
@@ -143,28 +186,58 @@ correct community identifier.
 [See example](https://zenodo.org/oai2d?verb=ListRecords&
 metadataPrefix=oai_datacite&set=user-cfa)
 
-If you need selective harvesting and your use case is not supported by above
-sets, please [contact us](http://about.zenodo.org/contact/) and we may possible
-create a specific set for you.
 
-## Update schedule
+```python
+# Fetch a couple of records from the OAI Set of the "cfa" community
+>>> records = sickle.ListRecords(metadataPrefix='oai_dc', set='user-cfa')
+>>> record = records.next()
 
-Most updates are available immediately, some few updates are only reflected in the OAI sets once an hour.
+# To inspect on what sets a record is in
+>>> record.header.setSpecs
+<...>
+```
 
-## Changes
+### Harvesting with a different metadata format
+There is also the possibility of using different metadata formats. For that, we only need to replace the `metadataPrefix` argument.
+
+[See example](https://zenodo.org/oai2d?verb=ListRecords&
+metadataPrefix=marc21&set=user-cfa)
 
-**2016-09-12**
+```python
+# Community harvest using "marc21" metadata format
+>>> records = sickle.ListRecords(metadataPrefix='marc21', set='user-cfa')
+>>> record = records.next()
+>>> record.metadata
+<...>
```

Comment:
```suggestion
<...>
```

I would provide a glimpse of what's available here, just to show that there's e.g. "title", "authors", "doi", etc.